src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -64,7 +64,7 @@ public EditCommand parse(String args) throws ParseException {
             throw new ParseException(EditCommand.MESSAGE_NOT_EDITED);
         }
 
-        return new EditCommand(index, editPersonDescriptor);
+        return new EditCommand(index, editPersonDescriptor); // more args here
```

Comment:
i think this commit can be for another PR since it is not related to the list test cases